### PR TITLE
Language fixes in the documentation.

### PR DESCRIPTION
**Pull request by asia-t**

### CODE REVIEW

File: src/closest_to/index.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ var parse = require('../parse/index.js')
  * @throws {TypeError} the second argument must be an instance of Array
  *
  * @example
- * // Which date is closer to 6 October 2015: 1 January 2000 or 1 January 2030?
+ * // Which date is closer to 6 September 2015: 1 January 2000 or 1 January 2030?
```
Comment (dgdavid) [inline src/closest_to/index.js]: `October` is the right month for `new Date(2015, 8, 6)`.

Comment (dgdavid) [inline src/closest_to/index.js]: Mmm, @LeshaKoss, @asia-t sorry. I meant to say `August`. But anyway, the @LeshaKoss' screenshot is a little weird for me.

Comment (LeshaKoss) [inline src/closest_to/index.js]: @dgdavid in JavaScript month indices (sadly) start with 0=January

Comment (dgdavid) [inline src/closest_to/index.js]: @LeshaKoss 

screen shot 2016-12-09 at 15 54 03

Comment (LeshaKoss) [inline src/closest_to/index.js]: @dgdavid `new Date(2015, 8, 6) !== new Date('2015, 8, 6')`

Also, weird, never thought of this way to create dates from strings 🤔

Comment (ertrzyiks) [inline src/closest_to/index.js]: @dgdavid it's August if you pass string to Date constructor but September if you pass numbers

Comment (dgdavid) [inline src/closest_to/index.js]: @LeshaKoss, @ertrzyiks ups! Sorry man! The quotation marks! My fault, I put it accidentally.

I had forgotten the [month's indices in JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date).

**Comment by LeshaKoss** [comment]
@asia-t  thanks again! Your fixes were released with [1.17.0](https://github.com/date-fns/date-fns/releases/tag/v1.17.0) (diff: https://github.com/date-fns/date-fns/compare/v1.16.0...v1.17.0) 🏆

**Pull request merged**